Create assessment group

markkramerus opened the pull request:
1) Modified the groupings (labvital -> assessments)
2) Added narrative on downloads page about the DD
3) Fixed a few bugs and added examples of MS

mlterryMitre reviewed (commented):
The assessment group changes look good, but there are other more substantial changes related to must-support in the profile conformance section that probably needs greater scrutiny.
If we need a quick merge, we at least need to note additional review is needed for conformance changes in this PR.

markkramerus commented:
To review:
1) Changed GenomicRegionStudies.value[x] to 0..0 to align with Genomics IG and address https://jira.hl7.org/browse/FHIR-31072
2) Add MS to RadiotherapySummary bodySite.extension to make sure extension is supported
3) Cleaned up ignoreWarnings.txt -- the build shows some warnings and broken links, but either they are legit or problems in the IGP
4) Cleaned up ComorbiditiesElixhauser profile page intro
5) Added re-grouping note to change log
6) Added some examples to conformance-profiles.md
7) Added narrative to download page -- probably not final, but a good start at least
8) Changed contents of group pages to reflect reorganization of groups
9) Change link https://hl7.org/fhir/us/core/2021Jan/StructureDefinition-us-core-vital-signs.html that was triggering warnings on the QA page to https://hl7.org/fhir/us/core/3.2.0/StructureDefinition-us-core-vital-signs.html -- but that didn't work. So either one is ok since they are legit links.
